src/test/ui/associated-type-bounds/assoc-type-eq-with-dyn-atb-fail.stderr

```diff
@@ -1,8 +1,8 @@
-error[E0277]: the trait bound `std::string::String: std::marker::Copy` is not satisfied
+error[E0277]: the trait bound `String: std::marker::Copy` is not satisfied
```

Comment:
The trait bound here isn't shorten.

Reply:
I'll correct this. This is because I treated all the different kind of `Namespace` as a single one, there are some places in `std` where you have both a macro and a type with the same same. This will be fixed.